packages/starlight/schemas/pagefind.ts

```diff
@@ -66,8 +66,11 @@ const pagefindIndexOptionsSchema = z.object({
 	language: z.string().optional(),
 	/**
 	 * Configure how search result rankings are calculated by Pagefind.
+	 * 
+	 * We applied the default value for the merged indexes in order to have the same search
+	 * conditions on them and do not let them have a better scoring than the current site.
 	 */
```

Comment:
I’m going to move this comment out of the JSDoc block because that text is something that is displayed to users in the editor and this feels more like an internal note to Starlight developers so we know why this was done in the future.

```suggestion
	 */
	 // We apply a default value to merged indexes in order to share the same ranking for them and the current site when not set explicitly.
```